subprojects/shipkit/src/main/groovy/org/shipkit/internal/notes/contributors/github/Contributors.java

```diff
@@ -15,9 +17,11 @@
      * @param apiUrl address of GitHub api endpoint, for example: "https://api.github.com"
      * @param repository name of GitHub repository, for example: "mockito/mockito"
      * @param readOnlyAuthToken the GitHub auth token
+     * @param ignoredContributors contributors to be ignored - VCS logins (e.g. 'shipkit-org')
```

Comment:
It's not consistent with documentation in ShipkitConfiguration.Git.getIgnoredContributors. Let's just use a reference to this method here.